lib/formatters/pretty.js

```diff
@@ -99,7 +102,10 @@ class PrettyPrinter {
       .map((error) => PrettyPrinter._formatError(error, options))
       .join('\n');
 
-    return `${chalk.underline(filePath)}\n${errorsMessages}\n`;
+    let displayFilePath =
```

Comment:
I know this is largely for CI only, but it'd be nice to have a test for this so we know what it's expected to look like in either case.

Reply:
Ya, totally right. I need to think on how to do that, might have to use a "DI" style injection to the constructor in a unit test to force both modes (since we actually do run in CI, the `ci-info` package will be `true` all the time so tests for the non-CI case will fail).